sig-data-science/charter.md

```diff
@@ -0,0 +1,43 @@
+# SIG Data Science Charter
+
+This charter adheres to the conventions described in the [Operate First Governance document](https://github.com/operate-first/community/blob/main/governance.md) and uses the Roles and Organization Management outlined in that document.
+
+## Scope
+
+SIG Data Science is responsible for all data science and machine learning related initiatives within the Operate First Community. This includes the development, exploration and communication of  cloud native machine learning tools and best practices, MLOps, as well as AIOps tools to enhance the services run by the open services group.
```

Comment:
For this suggestion, I'm also splitting the paragraph into a sentence per line to make the diff easier to read:

```suggestion
SIG Data Science is responsible for all data science and machine learning related initiatives within the Operate First community.
This includes the development, exploration, and communication of cloud native machine learning tools and best practices, MLOps, as well as AIOps tools to enhance the services run by the [Open Services Group](https://github.com/open-services-group/community/).
```